smb3parse/util/rom.py

```diff
@@ -46,6 +46,14 @@ def __init__(self, rom_data: bytearray, header: Optional[INESHeader] = None):
 
         self._header = header
 
+    @property
+    def header(self):
+        return self._header
+
+    @header.setter
+    def header(self, value):
+        self._header = value
+
```

Comment:
I don't think anyone really has the need to touch the header directly. Seems like the ROM should abstract that away and just have a property exposing `prg_units` and anything else the programmer might want to change about the ROM's properties.